src/Diagrams/Backend/SVG.hs

```diff
@@ -339,9 +346,54 @@ attributedRender svg = do
   clippedSvg   <- renderSvgWithClipping preT svg sty
   lineGradDefs <- lineTextureDefs sty
   fillGradDefs <- fillTextureDefs sty
+  let (transOriginAttr, transElement) = case getAttr sty of
+        Nothing -> ([], mempty)
+        Just (TransformAnimationAttribute animations t) ->
+          ( pure . makeAttribute "transform-origin" . (\(P (V2 x y)) -> showNum x <> " " <> showNum y) $ transform t 1
```

Comment:
This looks very slightly off for my working example (by about `V2 -3.3 3.3` for small circles, tending to 0 for larger ones). Not sure why. I guess I should come up with a simpler example anyway, to post in this thread as motivation.